EFAuditer/AuditDatabaseContext.cs

```diff
@@ -21,7 +21,7 @@ public async Task AuditOperationAsync(int primaryKeyId, string entity, string de
         private AuditLog CreateAuditLog(int primaryKeyId, string entity, string details, string eventType, string userName)
         {
             return new AuditLog() {
-                OriginalId = primaryKeyId,

```

Comment:
We're auditing the creation of new users now, which have email addresses as their primary key. Maybe they should be excluded from auditing, but for now just wanted to fix this.

Reply:
As I was running it from my dev environment, currently the audit user would have been me - not sure what would happen when running as a scheduled job, but I think null is a good bet!